### DISCUSSION TITLE
How to use in backend to return the formatted text in REST API

### DISCUSSION THREAD

**Discussion by SAGARSURI** [Textualize/rich#2836] [General]
I want to build a cli app and want to render markdown on terminal. I am using Dart language to build the cli app. I want to create an API which will process markdown texts using rich library and return the formatted text which I can directly print on the terminal. Is it possible?

**Comment by willmcgugan** [comment]
Rich can display Markdown in the terminal. I don't understand your other requirements. Rich works with Python, not Dart.

**Reply by SAGARSURI** [reply]
Sorry for the confusion, I mean is it possible to use rich library on the backend and return the formatted text through a REST API to a frontend app(written in some other language) to render it on terminal?

**Reply by willmcgugan** [reply]
You can export the output as HTML...

**Reply by SAGARSURI** [reply]
here frontend app is a cli app that can only be accessed through terminal
